services/ui-src/src/components/reports/ModalDrawerReportPage.tsx

```diff
@@ -178,11 +189,46 @@ export const ModalDrawerReportPage = ({ route, validateOnRender }: Props) => {
     headRow: ["", "", ""],
   };
 
+  const defaultsPopsNotSelected = (targetPopulations: AnyObject[]) => {
+    const defaultPopulationNames = getDefaultTargetPopulationNames();
```

Comment:
Probably gonna have to write some tests for this component's changes to make code climate happy 😭  Let me know if you want help with that...